Caboodle/Battery/Battery.shared.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Caboodle
+{
+    public static partial class Battery
+    {
+    }
+
+    public enum BatteryState
+    {
+        Unknown,
+        Charging,
+        Discharging,
+        Full,
+        NotCharging
+    }
+
+    public enum BatteryPowerSource
+    {
+        Unknown,
+        Battery,
+        AC,
+        USB,
```

Comment:
What are the rules for naming these? Always use Pascal case or do we allow acronyms all upper case? I know stylecop will complain.